test/functional/kernel/nested-loop/CMakeLists.txt

```diff
@@ -74,24 +64,24 @@ unset( NESTED_LOOPTYPES )
 #
 # If building a subset of openmp target tests, add tests to build here.
 #
-if(RAJA_ENABLE_TARGET_OPENMP)
-  if(RAJA_TEST_OPENMP_TARGET_SUBSET)
-
-    set(NESTED_LOOP_BACKEND OpenMPTarget)
-    set(NESTED_LOOPTYPES Basic MultiLambdaParam)
-
-    set(RESOURCE "-")
-    foreach( NESTED_LOOP_TYPE ${NESTED_LOOPTYPES} )
-      configure_file( test-kernel-nested-loop.cpp.in
-                      test-kernel-nested-loop-${NESTED_LOOP_TYPE}-${NESTED_LOOP_BACKEND}.cpp )
-      raja_add_test( NAME test-kernel-nested-loop-${NESTED_LOOP_TYPE}-${NESTED_LOOP_BACKEND}
-                     SOURCES ${CMAKE_CURRENT_BINARY_DIR}/test-kernel-nested-loop-${NESTED_LOOP_TYPE}-${NESTED_LOOP_BACKEND}.cpp )
-
-      target_include_directories(test-kernel-nested-loop-${NESTED_LOOP_TYPE}-${NESTED_LOOP_BACKEND}.exe
-                                 PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/tests)
-  endforeach()
-
-  endif()
-endif()
+##if(RAJA_ENABLE_TARGET_OPENMP)
```

Comment:
This one's commented out but many others are removed?

Reply:
Thanks. I missed that during my cleanup pass.